src/main/java/nl/tudelft/ti2206/group9/Main.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * 
+ */
+package nl.tudelft.ti2206.group9;
+
+import nl.tudelft.ti2206.group9.entities.AbstractEntity;
+import nl.tudelft.ti2206.group9.entities.Coin;
+import nl.tudelft.ti2206.group9.entities.Player;
+import nl.tudelft.ti2206.group9.level.State;
+import nl.tudelft.ti2206.group9.util.Point3D;
+
+/**
+ * @author Maarten
+ *
+ */
+public final class Main {
+
+	public static final int TRACKLENGTH = 50;
+	public static final int TICK = 100;
+	public static final double COINCHANCE = 0.07;
+
+	private static char[] track;
+
+	private Main() { }
+
+	/**
+	 * @param args
+	 * @throws IOException 
+	 * @throws InterruptedException 
+	 */
+	public static void main(String... args) throws InterruptedException {
+		State.resetAll();
+		State.getTrack().addEntity(new Coin(new Point3D(TRACKLENGTH, 0, 0)));
+
+		while (true) {
+			Thread.sleep(TICK);
```

Comment:
Nogmaals, dit was een quick-fix om het werkend te krijgen, maar threads gaan we zeker gebruiken :)